hyperjumptech/monika
# Fix: 228 mailgun vulnerability

# Monika Pull Request (PR)  
Fix #228 maigun vulnerability

## What feature/issue does this PR add  
Vulnerability have been highlighted by npm audit in dependency of mailgun-js

## How did you implement / how did you fix it  
1.  change library from mailgun-js to mailgun.js
2. add username as mailgun credentials to the configuration

## How to test  
1. create monika config with mailgun notifications :
```
{
      "id": "unique-id-mailgun",
      "type": "mailgun",
      "data": {
        "recipients": ["recipients@gmail.com"],
        "apiKey": "api-key",
        "domain": "sandbox.mailgun.org",
        "username": "postmaster@sandbox.mailgun.org"
      }
    }
```
2. run monika `./bin/run`
3. it should immediately send message to recipient mail
